src/app.js

```diff
@@ -1,13 +1,16 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import swagger from './routes/swagger';
+import userRoute from './routes/userRoute';
 
 const app = express();
 app.use(express.json());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+
 app.use('/API/v1', swagger);
```

Comment:
### Nitpick

- Maintain consistency with string casing, preferably should be of the lower case.